template/cordova/lib/PluginHandler.js

```diff
@@ -105,7 +105,7 @@ var handlers = {
             var type = obj.type;
 
             if(type === 'projectReference') {
-                project.removeProjectReference(path.join(plugin.dir, src), getTargetConditions(obj));
+                project.removeProjectReference(path.join('..', '..', 'plugins', plugin.id, src), getTargetConditions(obj));
```

Comment:
Updated, thanks for review!
There was an issue with mock dummyplugin, which PluginInfo.dir was initialized with a corresponding fixture path - so in case when the temp folder was on a different drive it caused path.relative to resolve in an absolute path in [PluginHandler.framework.install](https://github.com/daserge/cordova-windows/blob/2bc5d96bdd0f6c9328680bede1b535dde218e23c/template/cordova/lib/PluginHandler.js#L91).
Normally we have a plugin.dir path initialized with relative path to the project root, i.e. `project_root/plugins/plugin-id` - so I've updated the tests accordingly.
I've also tested that install/rm a plugin from another drive updates jsproj files properly.